.github/workflows/lighthouse-ci-prod.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   lhci:
+    if: github.repository == 'kawalcovid19/wargabantuwarga.com'
```

Comment:
What about the other `lighthouse-ci` file? Don't we need it configured there, Mas @rubiagatra?

What do you think, Mas @mazipan ?

Reply:
I don't know, need to test it.
I can test it in different PR.

Reply:
Wait let me double check

Reply:
Ah, why I don't include here @zainfathoni because the mindset is PR only to the upstream, not create PR to their own fork repo. If you want me to include it, it is no problem for me.

Reply:
it is only run when someone label `if: ${{ github.event.label.name == 'need-perf-check' }}`

Reply:
Oh, I see.. That makes sense. Okay, then. Let's go with this version for now, it should be good enough to go.

Thanks, Mas @rubiagatra!